tests/test_core.py

```diff
@@ -169,9 +178,18 @@ def test_executor():
             """Test executor."""
             call_count.append('call')
 
+        @asyncio.coroutine
+        def wait_finish_callback():
+            """Wait until all stuff is sheduled."""
+            while len(call_count) < 40:
```

Comment:
Any reason we pick such a big number? What advantage does it have over let's say, 2 callbacks?